File: 프로그래머스/unrated/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.java
class Solution {
    public int solution(int n) {
        
        if (n % 2 == 1)
            return (int)Math.pow((n + 1) / 2, 2);
        else
            return (n * (n + 1) * (n + 2) / 6);
        
    }
}

File: 프로그래머스/Lv.0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.js
function solution(n) {
    let sum = 0;
    if (n % 2 == 0)
        for (i = 2; i <= n; i += 2)
            sum += i * i;
    else
        for (i = 1; i <= n; i += 2)
            sum += i;
    return sum;
}